mempool/mempool.go

```diff
@@ -280,6 +287,7 @@ func (mem *Mempool) Flush() {
 
 	for e := mem.txs.Front(); e != nil; e = e.Next() {
 		mem.txs.Remove(e)
+		_ = atomic.SwapInt64(&mem.sizeBytes, 0)
```

Comment:
Shouldn't this be outside for loop? Just a side note: in this case strictly speaking we don't need atomic access as we hold `mem.proxyMtx.Lock()` but it's would help once we start getting rid of mempool locks. Maybe it's worth leaving a comment so reader is not confused.

Reply:
Good catch, thanks 👍